packages/bp/src/core/bots/bot-service.ts

```diff
@@ -326,9 +325,14 @@ export class BotService {
       } else {
         this.logger.forBot(botId).info(`Import of bot ${botId} was denied by hook validation`)
       }
+    } catch (error) {
```

Comment:
In the absence of a catch() statement, the error is bubbled-up, and it is catched by the `asyncMiddleware` in the `.post('/:botId/import')` statement, which will display the stack trace in the console and attach the ID of the bot to the log event. So, here it would not be necessary